styles.css

```diff
@@ -0,0 +1,593 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* NAVBAR STYLING */
+header {
+  position: absolute;
+  z-index: 2; /*  set index z value to the navbar not be overlaped by the floating adversing */
+}
+
+.navbar {
+  position: fixed;
+  display: flex;
+  width: 100%;
+  height: 50px;
+  justify-content: space-between;
+  align-items: center;
+  font-family: Arial, Helvetica, sans-serif;
+  border-bottom: #e2e2e2 solid 1px;
+  top: 0;
+  background-color: white;
+}
+
+.navbar-first-element button {
+  border: none;
+  background: none;
+  width: 25px;
+  height: 25px;
+  margin: 0 5px;
+}
+
+.navbar-third-element button {
+  background-color: #587697;
+  border: #3a5b93 1px solid;
+  color: white;
+  height: 30px;
+  padding: 0 15px;
+  border-radius: 3px;
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.navbar-third-element {
+  margin-right: 20px;
+}
+
+.link-space {
+  text-decoration: none;
+  color: black;
+  font-weight: bold;
+  font-size: 14px;
+}
+
+/* BOTTOM-NAVBAR STYLING */
+.nav-bottom {
+  font-family: Arial, Helvetica, sans-serif;
+  position: fixed;
+  bottom: 0;
+  display: block;
+  width: 100%;
+  height: 50px;
+  border-top: 1px solid black;
+  background-color: white;
+}
+
+.nav-bottom p {
+  margin-left: 35%;
+  margin-top: 15px;
+}
+
+.right-elem {
+  margin-top: 15px;
+  margin-right: 100px;
+  float: right;
+}
+
+.right-elem a {
+  margin-right: 10px;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+}
+
+.icon-arrow i {
+  color: black;
+}
+
+.clas {
+  display: inline-block;
+}
+
+/* MAIN SECTION STYLING */
+.out-container {
+  margin-top: 50px;
+  margin-bottom: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+.socialiAnchors {
+  color: black;
+}
+
+.container {
+  width: 956px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  font-family: 'Merriweather', serif;
+}
+
+.main-text {
+  font-size: 18px;
+  margin: 10px 150px;
+  line-height: 30px;
+}
+
+.container h2 {
+  margin: 30px 150px;
+  font-family: 'Balsamiq Sans', cursive;
+  font-size: 35px;
+}
+
+.image1 {
+  text-align: center;
+}
```

Comment:
This selector has the same styles as the selector on line 163. Group selectors to minimize code.